test/extended/machines/scale.go

```diff
@@ -186,6 +187,35 @@ func scaleMachineSet(name string, replicas int) error {
 }
 
 var _ = g.Describe("[sig-cluster-lifecycle][Feature:Machines][Serial] Managed cluster should", func() {
+
+	var (
+		c      *kubernetes.Clientset
+		dc     dynamic.Interface
+		helper *bmhelper.BaremetalTestHelper
+	)
+
+	g.BeforeEach(func() {
+
+		cfg, err := e2e.LoadConfig()
+		o.Expect(err).NotTo(o.HaveOccurred())
+		c, err = e2e.LoadClientset()
+		o.Expect(err).NotTo(o.HaveOccurred())
+		dc, err = dynamic.NewForConfig(cfg)
+		o.Expect(err).NotTo(o.HaveOccurred())
+
+		// For baremetal platforms, an extra worker must be previously
+		// deployed to allow subsequent scaling operations
+		helper = bmhelper.NewBaremetalTestHelper(dc)
+		if helper.CanDeployExtraWorkers() {
+			helper.Setup()
+			helper.DeployExtraWorker(0)
```

Comment:
This will only deploy one extra worker, is a mechanism needed to deploy more than one?